examples/machinelearningservices/computecluster.yaml

```
@@ -0,0 +1,191 @@
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputecluster
+    skuName: standard
+    tenantId: ${data.azurerm_client_config.current.tenant_id}
+---
+apiVersion: machinelearningservices.azure.upbound.io/v1beta1
+kind: Workspace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    applicationInsightsIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    identity:
+    - type: SystemAssigned
+    keyVaultIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: examplevault0001
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputecluster
+    storageAccountIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: exacc001
+    publicNetworkAccessEnabled: true
+---
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
+  labels:
+    testing.upbound.io/example-name: rgcomputecluster
+  name: rgcomputecluster
+spec:
+  forProvider:
+    location: West Europe
+---
+apiVersion: storage.azure.upbound.io/v1beta1
+kind: Account
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
+  labels:
+    testing.upbound.io/example-name: exacc001
+  name: exacc001
+spec:
+  forProvider:
+    accountReplicationType: LRS
+    accountTier: Standard
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputecluster
+---
+apiVersion: network.azure.upbound.io/v1beta1
+kind: Subnet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    addressPrefixes:
+    - 10.1.0.0/24
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputecluster
+    virtualNetworkNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+---
+apiVersion: network.azure.upbound.io/v1beta1
+kind: VirtualNetwork
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    addressSpace:
+    - 10.1.0.0/16
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgcomputecluster
+---
+apiVersion: alertsmanagement.azure.upbound.io/v1beta1
+kind: MonitorSmartDetectorAlertRule
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alertsmanagement/v1beta1/monitorsmartdetectoralertrule
```

```suggestion
    meta.upbound.io/example-id: machinelearningservices/v1beta1/computecluster
```

Could you please fix the example-id?